redi-connect-front/src/components/organisms/Navbar.scss

```diff
@@ -16,6 +16,10 @@
     }
   }
 
+  &__logo:hover {
+    text-decoration: none !important;
```

Comment:
Please leave a comment here about why this is being done. Useful to document for future developers.

Reply:
@ericbolikowski explanation added